scripts/db/schema.sql

```diff
@@ -0,0 +1,168 @@
+SET default_with_oids = false;
+
+CREATE TABLE canvas_site_mailing_list_members (
+    id SERIAL PRIMARY KEY,
+    mailing_list_id INTEGER NOT NULL,
+    email_address CHARACTER VARYING(255) NOT NULL,
+    can_send BOOLEAN DEFAULT FALSE NOT NULL,
+    first_name CHARACTER VARYING(255),
+    last_name CHARACTER VARYING(255),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    deleted_at TIMESTAMP WITH TIME ZONE,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    welcomed_at TIMESTAMP WITH TIME ZONE
+);
+ALTER TABLE ONLY canvas_site_mailing_list_members
+    ADD CONSTRAINT canvas_site_mailing_list_members_unique_constraint UNIQUE (mailing_list_id, email_address);
+CREATE INDEX canvas_site_mailing_list_members_deleted_at_idx
+    ON canvas_site_mailing_list_members USING btree (deleted_at);
+CREATE INDEX canvas_site_mailing_list_members_welcomed_at_idx
+    ON canvas_site_mailing_list_members USING btree (welcomed_at);
+
+--
+
+CREATE TABLE canvas_site_mailing_lists (
+    id SERIAL PRIMARY KEY,
+    canvas_site_id INTEGER NOT NULL,
+    canvas_site_name CHARACTER VARYING(255),
+    list_name CHARACTER VARYING(255),
+    members_count INTEGER,
+    populate_add_errors INTEGER,
+    populate_remove_errors INTEGER,
+    populated_at TIMESTAMP WITH TIME ZONE,
+    state CHARACTER VARYING(255),
+    type CHARACTER VARYING(255),
+    welcome_email_active BOOLEAN DEFAULT FALSE NOT NULL,
+    welcome_email_body TEXT,
+    welcome_email_subject TEXT,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+ALTER TABLE ONLY canvas_site_mailing_lists
+    ADD CONSTRAINT canvas_site_mailing_lists_unique_constraint UNIQUE (canvas_site_id);
+
+--
+
+CREATE TABLE canvas_synchronization (
+    id SERIAL PRIMARY KEY,
+    last_guest_user_sync TIMESTAMP WITH TIME ZONE,
+    latest_term_enrollment_csv_set TIMESTAMP WITH TIME ZONE,
+    last_enrollment_sync TIMESTAMP WITH TIME ZONE,
+    last_instructor_sync TIMESTAMP WITH TIME ZONE
+);
+
+--
+
+CREATE TABLE oauth2_data (
+    id SERIAL PRIMARY KEY,
+    app_id CHARACTER VARYING(255),
+    uid CHARACTER VARYING(255),
+    access_token TEXT,
+    refresh_token TEXT,
+    expiration_time BIGINT,
+    app_data TEXT
+);
+
+ALTER TABLE ONLY oauth2_data
+    ADD CONSTRAINT oauth2_data_unique_constraint UNIQUE (app_id, uid);
+
+--
+
+CREATE TABLE recent_uids (
+    id SERIAL PRIMARY KEY,
+    owner_id CHARACTER VARYING(255),
+    stored_uid CHARACTER VARYING(255),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE INDEX recent_uids_owner_id_idx ON recent_uids USING btree (owner_id);
+
+--
+
+CREATE TABLE saved_uids (
+    id SERIAL PRIMARY KEY,
+    owner_id CHARACTER VARYING(255),
+    stored_uid CHARACTER VARYING(255),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE INDEX saved_uids_owner_id_idx ON saved_uids USING btree (owner_id);
+
```

Comment:
@johncrossman Our snap decision to do without rebuilding view-as logic in Ripley means that we can get by without these recent_uids and saved_uids tables too.